dask_ml/model_selection/_split.py

```diff
@@ -150,17 +152,16 @@ def __init__(
 
     def split(self, X, y=None, groups=None):
         X = check_array(X)
-
+        rng = check_random_state(self.random_state)
         for i in range(self.n_splits):
+            seeds = draw_seed(rng, 0, 2 ** 32 - 1, size=len(X.chunks[0]), dtype="u8")
```

Comment:
Maybe `dtype='uint'` instead of `'u8`? I'm not sure what happens to `u8` on 32-bit platforms.